PSC-ONBOARDING-OFFBOARDING.md

```diff
@@ -0,0 +1,171 @@
+# On-boarding / Off-boarding Procedure for Product Security Committee Members
+
+This document outlines the process for individual(s) both joining or leaving
+the Kubernetes PSC (Product Security Committee).
+
+The key differentiator between on-boarding and off-boarding PSC members, is that
+individuals joining are first required to have completed the [Associate Role](#associate-role)
+before a nomination can be made from by any existing PSC members.
+
+For the purpose of example, we will use a placeholder name of **Jane Doe** with
+a github username **`jdoe`** and a company name of **ACME LTD**.
+
+### Associate Role
+
+Once a candidate has completed a 3 month rotation in the [Associate](https://github.com/kubernetes/security/blob/master/security-release-process.md#Associate) role, existing PSC Members can
+nominate said individual(s) for promotion from Associate to full PSC members. See
+[joining](https://github.com/kubernetes/security/blob/master/security-release-process.md#joining)
+for further details on the PSC selection process.
+
+This selection process is initiated with an email sent to the
+[kubernetes-security-discuss mailing list](https://groups.google.com/forum/#!forum/kubernetes-security-discuss)
+once the three month Associate Role period is complete.
+
+The following template may be used:
+
+```
+I want to nominate Jane Doe to join the Product Security Committee as
+a full member. She has been an associate member since December 4th 2018
+and in that role over the last three months he has helped on a number of
+tasks including:
+
+<list work carried out as associate>
+
+Our process for accepting this nomination is lazy consensus and I will
+close the round of consensus on March 15th, 2019 to leave time for any
+comments or question. If the nomination is accepted Jane will take on
+the full responsibilities of a Security Committee member and take on a
+weekly rotation for security@kubernetes.io on-call rotation.
+```
+
+### Required file and access grant updates.
+
+Once the PSC has promoted an individual(s)  to a full PSC member, various
+systems require updating. Likewise, should a PSC member leave the PSC - the
+reverse is required, with the user being removed from each system.
+
+#### kubernetes/community repository
+
+##### file: https://github.com/kubernetes/community/blob/master/sigs.yaml
+
+Add / remove the PSC member(s)  github name from `community/sigs.yaml` to the
+existing `label`| `leadership` field, according to the usernames alphabetical
+placing:
+
+```yaml
+label: product-security
+  leadership:
+    chairs:
+    - github: jdoe
+      name: Jane Doe
+      company: Acme LTD
+```
+
+##### file: https://github.com/kubernetes/community/blob/master/OWNERS_ALIASES
```

Comment:
This and the following files are all generated from the yaml above. Just update the yaml and run `make generate`

Reply:
Could you define `following files` please?

*  https://github.com/kubernetes/community/blob/master/OWNERS_ALIASES
* https://github.com/kubernetes/community/blob/master/committee-product-security/README.md
* file: https://github.com/kubernetes/community/blob/master/sig-list.md

Is that correct?

Reply:
Yep. An easy way to check is to just modify sigs.yaml, run the command, and see what changed.

Reply:
I think you can remove these sections from the guide, since no action is needed (other than the generate command already mentioned).